donor-portal/src/pages/about-us/styles.css

```diff
@@ -208,22 +213,26 @@
     margin: 1rem;
   }
 
-  .accountability-and-transparency-section, .profile-card-container,.partner-card-container,.partner-section-card,.volunteers-section{
-      flex: 1;
-      display: flex;
-      flex-direction: column;
-      align-items: center;
-      /* text-align: center; */
+  .accountability-and-transparency-section,
+  .profile-card-container,
+  .partner-card-container,
+  .partner-section-card,
+  .volunteers-section {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* text-align: center; */
   }
 
-  .profile-card{
+  .profile-card {
     text-align: center;
   }
 
-  .accountability-and-transparency, .paraghraph {
+  .accountability-and-transparency,
+  .paraghraph {
     padding-left: 1.5rem;
     padding-right: 1.5rem;
     text-align: justify !important;
   }
-  
-}
\ No newline at end of file
+}
```

Comment:
prettier changes.

Reply:
Thanks!
Couldn't help but notice the typo in _paragraph_ 🥲 Would u mind fixing that? @nipuna777

Reply:
Refactored this in the latest commit here: https://github.com/LSFLK/MedicinesforLK/pull/419/commits/9b7ac9d10b1932fecd1b51f993241a3514936085

I removed the `paraghraph` class and all references and opted instead for using the child combinator. This will allow us to select only the direct decedent paragraphs.

Reply:
Happy to do this is separate PR too.